tensorflow_serving/tools/pip_package/build_pip_package.sh

```diff
@@ -44,9 +46,20 @@ function main() {
 
   cp bazel-serving/tensorflow_serving/apis/*_grpc.py \
     "${TMPDIR}/tensorflow_serving/apis"
+  
+  cp bazel-genfiles/tensorflow_serving/util/*_pb2.py \
+    "${TMPDIR}/tensorflow_serving/util"
+
+  cp bazel-genfiles/external/org_tensorflow/tensorflow/core/lib/core/*.py \
```

Comment:
What's being added from here?

Reply:
The status proto depends on tensorflow error_codes proto the same as py codes.

Ref:
- https://github.com/tensorflow/serving/blob/1.9.0/tensorflow_serving/apis/get_model_status.proto#L6
- https://github.com/tensorflow/serving/blob/1.9.0/tensorflow_serving/util/status.proto#L5
- https://github.com/tensorflow/tensorflow/blob/v1.9.0/tensorflow/core/lib/core/error_codes.proto

Reply:
@gautamvasudevan The status proto depends on tensorflow error_codes proto the same as py codes.

Ref:
- https://github.com/tensorflow/serving/blob/1.9.0/tensorflow_serving/apis/get_model_status.proto#L6
- https://github.com/tensorflow/serving/blob/1.9.0/tensorflow_serving/util/status.proto#L5
- https://github.com/tensorflow/tensorflow/blob/v1.9.0/tensorflow/core/lib/core/error_codes.proto

Reply:
Sounds good.